nesta/packages/quality_check/quality_calculations.py

```diff
@@ -0,0 +1,68 @@
+import pandas as pd
+import itertools
+
+def missing_values(data_frame):
+
+    """
+
+    """
+    return data_frame.isnull().sum().sort_values(ascending=False)
+
+def missing_value_column_count(data_frame):
+

```

Comment:
Might be a bit more readable to use numpy's `histogram` function

```suggestion
data_frame_missing = data_frame.isnull().sum() / data_frame.shape[0]
out = np.histogram(data_frame_missing, bins=range(0, 101, 10))
return out
```